compiler/rustc_session/src/options.rs

```diff
@@ -1714,6 +1714,8 @@ options! {
         "emit the bc module with thin LTO info (default: yes)"),
     enforce_type_length_limit: bool = (false, parse_bool, [TRACKED],
         "enforce the type length limit when monomorphizing instances in codegen"),
+    error_metrics: Option<PathBuf> = (None, parse_opt_pathbuf, [UNTRACKED],
```

Comment:
Lets rename this flag to something more general, like just `metrics` or `metrics_directory`, as it will be used for more things beyond "errors".